screens/createEventPage.js

```diff
@@ -141,7 +141,7 @@ class eventMapPage extends Component{
               <Text style={styles.OBtext}>EVENT</Text>
           </TouchableOpacity>
         </View>
-        <View style={{height: 80}}/>
```

Comment:
I would also add a scrollview to this as well because the createEvent might be out of scope for the page.